### ISSUE TITLE
Not working in Blender 3.0

### ISSUE THREAD

**Issue by wrthr** [neverhood311/Stop-motion-OBJ#153]
**System Information**
Operating system: Windows 10 Pro

**Addon version**
Broken: 2.1.1
Worked: 2.1.1 - but with blender 2.8

**Describe the bug**
After upgrading to Blender 3, import mesh disappeared as an option, and I can't seem to even reinstall the add on. After attempting to install the add on, the add on does not appear in the list of add ons, so I can't enable it.

**Steps To Reproduce**
Try installing this add on in Blender 3.

**Expected behavior**
I expected the add on to work in Blender 3.0.
Failing that, I expected to be able to reinstall the add on in Blender 3.0.

**Comment by neverhood311** [comment]
This is unusual. Mine worked fine, both when I upgraded from 2.93.6 to 3.0 and when I uninstalled then reinstalled the addon in 3.0.

You might need to manually remove the addon from the file system, then reinstall it from within Blender 3.0.

On Windows, the addon should be installed in: `C:\Users\<username>\AppData\Roaming\Blender Foundation\Blender\3.0\scripts\addons`. It should be in a folder called `Stop-motion-OBJ` (if it's not there, try looking for a folder called `src`, which was the case on my machine). Delete that folder. Then open up Blender and install the addon from the Edit > Preferences window.
